data/json/items/comestibles/alcohol.json

```
@@ -1418,7 +1418,7 @@
     "phase": "liquid",
     "charges": 6,
     "flags": [ "EATEN_COLD", "EDIBLE_FROZEN", "NO_AUTO_CONSUME" ],
-    "fun": -12,
+    "fun": 12,
```

```suggestion
    "fun": 12,
```
Ooof even the description is weird.  So I think if we wanted to do something like this http://mind-over-batter.com/boozy-drinks-2/blueberry-infused-vodka/ it would make sense to have decent fun.  But looking at the recipe this isn't anything like that just berries boiled into a mush in vodka.  I would think it would have a -2 or a plus 2 but really no more than that just due to strange consistency.

---

+2 sounds ok, it wouldn't make your mood go downhill considering it's made of alcohol and berries.

---

sounds good